3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/utils/shredder/ShredderSpec.scala

```diff
@@ -17,22 +17,17 @@ package common
 package utils
 package shredder
 
-// Snowplow Utils
-import util.Tap._
-
 // Snowplow Common Enrich
 import outputs.EnrichedEvent
 
 // Specs2
 import org.specs2.Specification
 
-class ShredderSpec extends Specification /*with DataTables with ValidationMatchers*/ { def is =
-
-  "This is a specification to test the Shredder functionality"                       ^
-                                                                                    p^
-  "makePartialHierarchy should initialize a partial TypeHierarchy"                   ! e1^
-  "shred should extract the JSONs from an unstructured event with multiple contexts" ! e2^
-                                                                                     end
+class ShredderSpec extends Specification /*with DataTables with ValidationMatchers*/ { def is = s2"""
```

Comment:
it was you! :laughing: https://github.com/snowplow/snowplow/blame/937bd51ad2f756462f46584afee4e369ae82b7a8/3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/utils/shredder/ShredderSpec.scala